package.json

```diff
@@ -13,6 +13,7 @@
 	},
 	"workspaces": [
 		"public_html/wp-content/mu-plugins/blocks",
+		"public_html/wp-content/mu-plugins/virtual-embeds",
```

Comment:
Why create a separate project if these are also blocks?

Reply:
These seemed conceptually different enough, not dealing with the WC content or APIs or anything; and because of that I didn't need the whole file/classes infrastructure of the blocks project.